thunder/tests/opinfos.py

```diff
@@ -8269,7 +8269,7 @@ def scaled_dot_product_attention_sample_generator(op, device, dtype, requires_gr
 
     # mask cases
     q, k, v = make(N, n_head, L, E), make(N, n_head, S, E), make(N, n_head, S, Ev)
-    bool_attn_mask = make((L, S), dtype=torch.bool, low=1, high=1, requires_grad=False).tril()
+    bool_attn_mask = torch.ones((L, S), dtype=torch.bool, requires_grad=False).tril()
```

Comment:
The test errors are real because this call to `torch.ones` doesn't pass a `device` argument like `make` does